NOTICE

```diff
@@ -4,13 +4,18 @@ You may not use the component except in compliance with the License.
 
 The component may include material which is licensed under other licenses / copyrights as
 listed below. Your use of this material within the component is also subject to the terms and
-conditions of these licenses. The LICENSE file contains the text of all the licenses which apply within this component, except as indicated in the following paragraph.
+conditions of these licenses. The LICENSE file contains the text of all the licenses which apply within this component, except as indicated in the following paragraphs.
 
 The static/fonts/JuliusSansOne directory within this component contains fonts which are licensed
 under the SIL Open Font License, version 1.1.  The static/fonts/JuliusSansOne directory contains its
 own license file, called LICENSE, which describes the licensing of the material
 in the static/fonts/JuliusSansOne directory.
 
+The static/images directory within this component contains images which are
+licensed under the ODbL v1.0 License.  The static/openstreetmap directory contains its
+own license file, called LICENSE, which describes the licensing of the material in the
+static/openstreetmap directory.
+
 Roboto Regular font is:
 Copyright 2017 Google Inc. All Rights Reserved.
 Licensed under the Apache License, Version 2.0
```

Comment:
My mistake: please delete lines 23-end as they should be covered in the separate license.